core/lib/libtomcrypt/src/pk/x25519/x25519_make_key.c

```diff
@@ -27,7 +27,6 @@ int x25519_make_key(prng_state *prng, int wprng, curve25519_key *key)
 {
    int err;
 
-   LTC_ARGCHK(prng != NULL);
```

Comment:
@etienne-lms previous discussion: https://github.com/OP-TEE/optee_os/pull/5297#discussion_r864622745

Reply:
> @etienne-lms previous discussion: [#5297 (comment)](https://github.com/OP-TEE/optee_os/pull/5297#discussion_r864622745)

@sohhass I believe the reason why the check is removed needs to be better documented. Please create a separate commit. It will also minimize the risk of this line being inadvertently dropped if a conflict occurs when rebasing to a more recent version of LibTomCrypt.

Reply:
Right, sorry, i didn't look enough in the P-R history.